README.md

```diff
@@ -3616,6 +3616,9 @@ resource cleanup when possible.
 * <a name="prefer-public-send"></a>
   Prefer `public_send` over `send` so as not to circumvent `private`/`protected` visibility.
 <sup>[[link](#prefer-public-send)]</sup>
+* <a name="prefer-__send__"></a>
+  Prefer `__send__` over `send` as `send` may overlap with existing methods.
```

Comment:
The rule should be a bit longer - some explanation for it is in order. You should also add a few code snippets to illustrate it.